sdk/src/test/java/com/mapbox/maps/StyleObserverTest.kt

```diff
@@ -99,8 +99,8 @@ class StyleObserverTest {
     val styleObserver = StyleObserver(mockk(relaxed = true), mockk(relaxed = true), 1.0f)
     val errorListener = mockk<OnMapLoadErrorListener>(relaxed = true)
     styleObserver.onNewStyleLoad(mockk(relaxed = true), errorListener)
-    styleObserver.onMapLoadError(MapLoadErrorType.GLYPHS, "foobar")
-    verify { errorListener.onMapLoadError(MapLoadErrorType.GLYPHS, "foobar") }
+    styleObserver.onMapLoadError(MapLoadErrorType.GLYPHS, "foobar", null, null)
```

Comment:
shouldn't it be better to check non-null values? to make sure we did not mess up smth

Reply:
it doesn't really matter as we just verify the same parameters are called on the listener.